Libraries/LibWeb/MediaCapabilitiesAPI/MediaCapabilities.h

```diff
@@ -1,15 +1,104 @@
 /*
  * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ * Copyright (c) 2025, Psychpsyo <psychpsyo@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
 #include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/EncryptedMediaExtensions/EncryptedMediaExtensions.h>
 
 namespace Web::MediaCapabilitiesAPI {
 
+// https://w3c.github.io/media-capabilities/#dictdef-videoconfiguration
+struct VideoConfiguration {
+    String content_type;
+    unsigned long width;
+    unsigned long height;
+    Optional<unsigned long> bitrate;
+    double framerate;
+    Optional<bool> has_alpha_channel;
+    Bindings::HdrMetadataType hdr_metadata_type;
+    Bindings::ColorGamut color_gamut;
+    Bindings::TransferFunction transfer_function;
```

Comment:
I don't believe it is correct to mark this as always required. It is not marked as a required field in the IDL, so is only optionally set in the prototype. Right now, this is falling back to the default initialization of the enum, but I believe it should instead be Optional (here, and for the other non-required dictionary fields).

I would personally also be more comfortable with always initializing these fields as well. If it weren't for the IDL prototype doing an explicit value initialization, this would result in reading from an unset variable.

Reply:
Done.